Jamfile

```diff
@@ -27,7 +27,6 @@ if $(BOOST_ROOT)
 else
 {
 	local boost-lib-search-path =
-		<search>/usr/lib
```

Comment:
To rephrase the above: If you use a custom toolchain or specify --sysroot, then explicitly passing `/usr/lib` is a problem, if you use a system toolchain, then it takes higher priority.